src/guides/village/strongman.mdx

```diff
@@ -6,11 +6,26 @@ route: /roles/Strongman
 
 # Strongman
 
-The Strongman can cause a village win if they are the only village member left alive againt a single evil. Otherwise, they act as a normal villager.
+The Strongman is a member of the Village. If they are the sole surviving village member left alive against a single evil, they cause a village win. They are otherwise just like a normal villager.
 
 ## Role Type
 
 - The Strongman is seen as member of the village by the Seer.
 - The Strongman is not seen as a user of witchcraft.
 - The Strongman is not seen visiting by the Harlot/Stalker/Familiar.
 - The Strongman is not seen as a killer by the Adjudicator.
+
+## Notes
+
+- If there are two players left in the game, and the Strongman is one of them, the game will award the victory to the village. The faction or role of the other player is irrelevant.
+- The Strongman is usually only found in speed games, where it is more likely to reach a final-two situation.
+
+## Tips
+
+- The Strongman is one of the few village roles where it may be more beneficial to lie about having the role than to be truthful. If any evils are made aware of a Strongman, they will want to kill them to ensure that they cannot lose in a one-versus-one scenario at the end of the game. This is especially the case for a solo evil like the Skinchanger.
+
+## History
+
+<Timeline lineColor="white">
+  <Event interval="2018-12-xx">Introduced.</Event>
```

Comment:
Does this work? I'd expect it to need an actual date.

Reply:
I've updated it to match the first reference to the role being in a game on Discord. It may not be entirely accurate, but sans knowing precisely which game it was first introduced in, it is the best solution.

The date is now 2018-12-08. If it was actually introduced before this it can be changed once more, but it probably isn't really necessary to do so.